static/js/hotkey.js

```diff
@@ -55,7 +55,12 @@ const keydown_cmd_or_ctrl_mappings = {
     190: {name: 'narrow_to_compose_target', message_view_only: true}, // '.'
 };
 
-const keydown_either_mappings = {
+
+var keydown_ctrl_and_shift_mappings = {
```

Comment:
This should be a `const` if I'm not wrong. We've stopped using vars anymore (and the existing ones are already being changed).